nimble/host/include/host/ble_gap.h

```diff
@@ -965,6 +965,7 @@ int ble_gap_ext_adv_stop(uint8_t instance);
 int ble_gap_ext_adv_set_data(uint8_t instance, struct os_mbuf *data);
 int ble_gap_ext_adv_rsp_set_data(uint8_t instance, struct os_mbuf *data);
 int ble_gap_ext_adv_remove(uint8_t instance);
+int ble_gap_ext_adv_clear(void);
```

Comment:
Could you please add doxygen for this new API? I know that there is a bad example above with missing doxygen and for that I have just created an issue on Nimble, so don't worry about that.

Reply:
Added.

Reply:
thanks, we just miss `@return ` with error code explanation.

Reply:
Added.